nss-day-cohort-60/rare-client-tenacious-tators
# Lh comments

Closes Ticket #122


# Description of Proposed Changes
Added functionality to update comments for posts
Also added author to Comment.js
---


# Steps to Test

1. Fetch the lh-comments branch and switch to it
2. Make sure that you have fetched lh-comments for server as well 
3. Navigate to My Posts and click View Comments
4. Click Edit & make edits. 
5. When you click Publish, verify that you are navigated back to the comments & that your changes have been saved.
